interfaces/apparmor/apparmor.go

```diff
@@ -83,7 +82,10 @@ func maybeSetNumberOfJobs() string {
 			return fmt.Sprintf("-j%d", cpus-2)
 		}
 	}
-	return ""
+	// We have a single CPU, which could be beefy, but likely isn't, pass
+	// -j0 which has a special meaning so that the main apparmor_parser
+	// process is used for compilation.
+	return "-j0"
```

Comment:
minor: not sure what is the most go-ish way to do this (switch or set of if/else...), but it would seem like these three cases could be handled at the same level:
```golang
if cpus <= 1 {
    ...
} else if cpus == 2 {
    ...
} else { // cpus > 2
    ...
}
```

Reply:
I fully agree with this proposal, the code as-is is a bit too deeply nested for my tastes :smiley: